clippy_lints/src/methods/mod.rs

```diff
@@ -330,6 +330,24 @@ declare_clippy_lint! {
     "using `Option.map_or(None, f)`, which is more succinctly expressed as `and_then(f)`"
 }
 
+declare_clippy_lint! {
+    /// **What it does:** Checks for usage of `_.map_or(None, Some)`.
+    ///
+    /// **Why is this bad?** Readability, this can be written more concisely as
+    /// `_.ok()`.
+    ///
+    /// **Known problems:**
```

Comment:
NIT:
```suggestion
    /// **Known problems:** None.
```

Reply:
Fixed in 91759a7582a292c3f85204b1d61e7185c8543959